### DISCUSSION TITLE
YOLOv8 - Model Ensembling

### DISCUSSION THREAD

**Discussion by dylacaru** [ultralytics/ultralytics#654] [Q&A]
Hello,

Does anyone happen to know if model ensembling is possible on the new YOLOv8?

Thanks in advance,

**Comment by moahaimen** [comment]
as i read somewhere no it doesnt work, and also i have the same question too

**Reply by dylacaru** [reply]
Thank you for your reply.
Hopefully this will be fixed in the near future.

**Reply by moahaimen** [reply]
you welcome, but if it happen please  tell me 
thank you

**Reply by glenn-jocher** [reply]
@moahaimen hello there! 👋

As of now, model ensembling is not a built-in feature of YOLOv8. However, the YOLO community is always evolving, and we appreciate your interest in such advanced functionalities. Keep an eye on our updates and documentation for any new features that may be introduced in the future. Your feedback is valuable to us, and it helps guide the development of our models. Thank you for being part of the YOLO community! 🚀

For the latest information and best practices, please refer to our documentation at https://docs.ultralytics.com.
